rclpy/src/rclpy/subscription.hpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLPY__SUBSCRIPTION_HPP_
+#define RCLPY__SUBSCRIPTION_HPP_
+
+#include <pybind11/pybind11.h>
+
+#include <string>
+
+namespace py = pybind11;
+
+namespace rclpy
+{
+/// Create a subscription
+/**
+ * This function will create a subscription for the given topic name.
+ * This subscription will use the typesupport defined in the message module
+ * provided as pymsg_type to send messages over the wire.
+ *
+ * On a successful call a list with two elements is returned:
```

Comment:
I think this return type description is outdated. Only a capsule is returned.

Reply:
Gah, I updated the `\return` below, but missed this mention. Good catch.

Reply:
Fixed in f9692bb.